WooCommerce/Classes/ViewRelated/Dashboard/MyStore/TopPerformerDataViewController.swift

```diff
@@ -146,7 +146,6 @@ private extension TopPerformerDataViewController {
     func configureTableView() {
         tableView.backgroundColor = .basicBackground
         tableView.separatorColor = .systemColor(.separator)
-        tableView.allowsSelection = false
```

Comment:
I wonder why it was set to false before 🤔

Reply:
Good question, I don't know why. But the problem is that not only had it been disabled here, but also from the storyboard, therefore you weren't able to select cells inside the table view.
I do a lot of tests and everything works correctly now 🤷‍♂